git-webhook/Makefile

```diff
@@ -0,0 +1,36 @@
+BINDIR           ?= $(CURDIR)/bin
+
+# Run tests
+test:
+	$(BINDIR)/ginkgo \
+		--randomizeAllSpecs --randomizeSuites --failOnPending \
+		--cover --coverprofile cover.out --trace --race \
+		./pkg/... ./cmd/...
+
+# Build dashboard binary
+build: fmt vet
+	go build -o bin/dashboard .
+
+# Run against the configured Kubernetes cluster in ~/.kube/config
+run: fmt vet
+	go run ./main.go
+
+# Run go fmt against code
+fmt:
+	go fmt ./pkg/... ./cmd/...
+
+# Run go vet against code
+vet:
+	go vet ./pkg/... ./cmd/...
+
+lint:
+	GOGC=off $(BINDIR)/golangci-lint run ./pkg/... ./cmd/...
+
+debug:
+	pwd
+
+# Installs necessary dependencies to run and test the project
+dependencies:
```

Comment:
You should install dependencies in the top-level makefile. You can switch the image in `.drone.yaml` to bfc.